### ISSUE TITLE
Add a net8.0 target

### ISSUE THREAD

**Issue by m-redding** [openai/openai-dotnet#272]
### Confirm this is a feature request for the .NET library and not the underlying OpenAI API

- [X] This is a feature request for the .NET library

### Describe the feature or improvement you are requesting

It would be great if OpenAI could add a net8.0 target, in order to unblock adding a net8.0 target to Azure.AI.OpenAI. There are more details in this issue: https://github.com/Azure/azure-sdk-for-net/issues/46891

Alternatively, resolving https://github.com/openai/openai-dotnet/issues/259 would also unblock adding this target to Azure.AI.OpenAI

**Comment by Frulfump** [comment]
Target was added in https://github.com/openai/openai-dotnet/pull/277 so available since OpenAI_2.2.0-beta.1 (no GA release yet)

**Comment by m-redding** [comment]
Yeah looks like this was fixed at some point since Azure OpenAI was able to add the net8.0 target https://github.com/Azure/azure-sdk-for-net/pull/48350
